app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java

```diff
@@ -323,10 +325,10 @@ protected void setMediaPreviews(final List<Attachment> attachments, boolean sens
             mediaPreviews[i].setVisibility(View.VISIBLE);
 
             if (TextUtils.isEmpty(previewUrl)) {
-                Picasso.with(context)
+                Glide.with(mediaPreviews[i])
                         .load(mediaPreviewUnloadedId)
-                        .resize(maxW, maxH)
-                        .onlyScaleDown()
+                        .override(maxW,maxH)
+                        .downsample(DownsampleStrategy.CENTER_INSIDE)
```

Comment:
This downsampling was used to prevent a crash when loading huge images. It looks like Glide is already resizing to the size of the ImageView, no need to help it

Reply:
I've removed downsampling